CIP-54: Community rewards go to reserve if undercollaterized (#320)
celo-org/celo-proposals: CIPs/cip-communit-rewards-reserve.md

---
cip: <to be assigned>
title: Community rewards go to reserve if undercollaterized
author: Paul Lange (@palango)
discussions-to: <URL>
status: Draft
type: Standards Track
category: Ring 0
created: 2023-03-03
license: Apache 2.0
---

## Simple Summary

Simplify the epoch rewards by never sending funds designated for the community fund to the stable coin reserve instead.

## Abstract

> A part of the epoch rewards is sent to the community fund. However, in the case of the stable coin reserve being undercollaterized, those funds are sent to the reserve instead. This CIP proposes to never send funds designated for the community fund to the reserve.

Reviewer: ```suggestion
A part of the epoch rewards is sent to the community fund. However, in the case of the stable coin reserve being undercollateralized, those funds are sent to the reserve instead. This CIP proposes to never send funds designated for the community fund to the reserve.
```

## Motivation

> The stable coins, and with them the underlying reserve, are now maintained by Mento. As an independent entity it should not directly receive funds designated for the community fund.

Reviewer: ```suggestion
The stable coins, and with them the underlying reserve, are now maintained by Mento. As an independent entity, it should not directly receive funds designated for the community fund.
```

Additionally the protocol gets simplified by reducing the number of conditional actions.

## Specification

> During the distribution of epoch rewards it will not be checked if the stable coin reserve is undercollaterized. Instead, funds designated for the community fund will always be send there.

Reviewer: ```suggestion
During the distribution of epoch, rewards it will not be checked if the stable coin reserve is undercollateralized. Instead, funds designated for the community fund will always be sent there.
```
